### DISCUSSION TITLE
Windows 10 VM

### DISCUSSION THREAD

**Discussion by fguisso** [tteck/Proxmox#1283] [Ideas]
Does anyone have these scripts for starting up a Win10 VM?

I'm using proxmox for little services and sometimes I like to use a win10 to play some games.

**Comment by platini76** [comment]
I need dor windows server or windows 11.
I want to execute crysral disk to see disk performance. any help?

**Comment by asian-boyfriend** [comment]
Pumping this to get some assistance. Thank you
